packages/core/src/queries/sso-connectors.ts

```diff
@@ -9,11 +11,42 @@ import { sql, type CommonQueryMethods } from '@silverhand/slonik';
 import SchemaQueries from '#src/utils/SchemaQueries.js';
 import { convertToIdentifiers } from '#src/utils/sql.js';
 
+import { buildInsertIntoWithPool } from '../database/insert-into.js';
+import { buildUpdateWhereWithPool } from '../database/update-where.js';
+
+const {
+  table: SsoConnectorIdpInitiatedAuthConfigsTable,
+  fields: SsoConnectorIdpInitiatedAuthConfigsFields,
```

Comment:
nit: should use camel case instead of pascal case?

Reply:
good catch